app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> int:
+        if args in cache_dict:
+            print("Getting from cache")
+        else:
+            print("Calculating new result")
+            cache_dict[args] = func(*args)
```

Comment:
The function receives both args and kwargs as input. And kwargs should also be stored in the cache.
```suggestion
            cache_dict[args] = func(*args, **kwargs)
```